lvm.go

```diff
@@ -61,16 +61,16 @@ type VolumeManager interface {
 // block device or other disk like devices that provide storage capacity.
 type PV struct {
 	// Name returns the name of the PV.
-	Name string ``
+	Name string `json:"name"`
 
 	// Path returns the device path of the PV.
-	Path string
+	Path string `json:"path"`
 
 	// Size returns the size of the PV.
-	Size uint64
+	Size uint64 `json:"size"`
 
 	// FreeSize returns the free size of the PV.
-	FreeSize uint64
+	FreeSize uint64 `json:"freeSize"`
```

Comment:
This seems probably better as a method as the freeSize is going to change. no?

Reply:
Yes. Let me fix it in the next PR.

Reply:
I think don't bother with that right now, but it may need fixing or we're going to re-scan a lot to get updated structures.

The same is true with VG , and it has both Size and FreeSize which are dynamic.